templates/base.html

```diff
@@ -7,8 +7,7 @@
             font-family: Arial, Helvetica, sans-serif;
             background-color: #efefef;
             width: 100%;
-            height:100%;
-            overflow: hidden;
+            height: 100%;
```

Comment:
The autonomous car instructions page is a bit larger due to new description and instructions sections as well as now offering multiple endpoints for level 1. 

`overflow: hidden;` would make scrolling down impossible.